tests/unit/test_messages_helpers.py

```diff
@@ -127,3 +129,46 @@ def test_getlogpath_ignore_other_files(test_log_dir, monkeypatch):
     new_fpath.touch()
     present_logs = sorted((test_log_dir / "testapp").iterdir())
     assert present_logs == [f_aaa] + previous_fpaths[1:] + [new_fpath, f_zzz]
+
+
+# -- tests for the _Progresser class
+
+
+def test_progresser_absolute_mode():
+    """Just use _Progresser as a context manager in absolute mode."""
+    stream = sys.stdout
+    text = "test text"
+    total = 123
+    fake_printer = MagicMock()
+    with _Progresser(fake_printer, total, text, stream, delta=False) as progresser:
+        progresser.advance(20)
+        progresser.advance(30)
```

Comment:
I think we could also test for out of range values, and also floating point numbers since we allow floats.

Reply:
Good idea about floating point. Regarding "out of range", we're not doing anything special with them (the code that actually *shows* the progress deals with that)... what kind of test are you suggesting? Just that it doesn't crash?

Reply:
Yes, to make sure it doesn't crash.